ideas/172-topic-democracy.md

```diff
@@ -0,0 +1,83 @@
+# Democracy
+
+## Preamble
+
+    Idea: 172-topic-democracy
+    Title: Subjective Delegation Liquid Democracy
+    Status: Draft
+    Created: 2018-04-15
+
+## Summary
+A democracy where you can specify delegate for carbon-voting specific multi level topics or fallback to a predefined defaults if none set.
```

Comment:
What do you mean by 'multi level topics'?

Reply:
I'll better detail this.